package.json

```diff
@@ -17,32 +17,15 @@
     "clean": "lerna run clean --parallel --no-bail",
     "test": "lerna run --stream --concurrency 1 test",
     "test:ci-non-playground": "lerna run --ignore @counterfactual/playground --ignore @counterfactual/dapp-high-roller --stream test",
-    "ganache": "ganache-cli --networkId ${npm_package_config_ganacheNetworkID} --verbose --gasLimit ${npm_package_config_ganacheGasLimit} --gasPrice ${npm_package_config_ganacheGasPrice} --port ${npm_package_config_ganachePort} --deterministic --account=\"${npm_package_config_unlockedAccount0},${npm_package_config_etherBalance}\" --account=\"${npm_package_config_unlockedAccount1},${npm_package_config_etherBalance}\" --account=\"${npm_package_config_unlockedAccount2},${npm_package_config_etherBalance}\" --account=\"${npm_package_config_unlockedAccount3},${npm_package_config_etherBalance}\" &> /dev/null &",
```

Comment:
do we not get top-level ganache anymore? how do some of the machine tests which run against an EVM work? can you reflect these in the README and circleci?

Reply:
Is the rewrite part of this PR? It was a pretty useful test

Reply:
I can add one back in but I think it would be better to do it in a new PR

Reply:
why can't they just be added back?

Reply:
@snario that's a pretty useful test (as @IIIIllllIIIIllllIIIIllllIIIIllllIIIIll mentioned) - let's either not remove it or make a tangible PR to add it back that's referenced here, so as to not lose track of it.